src/index.ts

```diff
@@ -215,10 +215,31 @@ if (ENABLE_GRAPHQL_UPLOAD) {
 }
 
 app.use("/v2", (req, res, next) => {
-  if (req.url !== "/") return next()
+  // This matches all routes that start with '/v2'.
+  //
+  // We only want to support:
+  //   - POSTs -> requests to '/v2'
+  //   - GETs:
+  //     - requests to '/v2' (loads GraphiQL in non-prod mode)
+  //     - requests to '/v2?query=...' (runs a query)
+  if (req.method === "POST") {
+    if (req.url !== "/") return next()
```

Comment:
This is so we don't 'silently' accept a path like "/v2/api/v1". When you do a POST, it must be at the root ("/v2").